core/src/main/java/hudson/model/Actionable.java

```diff
@@ -95,7 +102,15 @@
         for (TransientActionFactory<?> taf : ExtensionList.lookup(TransientActionFactory.class)) {
             if (taf.type().isInstance(this)) {
                 try {
-                    _actions.addAll(createFor(taf));
+                    Collection<? extends Action> newActions = createFor(taf);
+                    for (Action action : newActions) {
+                        if (action instanceof Action) {
+                            _actions.add(action);
+                        } else {
+                            LOGGER.log(Level.WARNING, "Actionable#getAllActions: Found a not action: " + action.getClass().toString());
```

Comment:
The most interesting information, `taf.getClass().getName()` (i.e., the culprit) is not even included in this message.